digitalocean/import_digitalocean_kubernetes_node_pool_test.go

```diff
@@ -0,0 +1,68 @@
+package digitalocean
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccDigitalOceanKubernetesNodePool_Import(t *testing.T) {
```

Comment:
@andrewsomething: This test fails as follows:

```
tdyas@mirage:~/Projects/Infra/terraform-provider-digitalocean$ TF_LOG=DEBUG TF_LOG_PATH=./log.out make testacc TESTARGS='-run=TestAccDigitalOceanKubernetesNodePool_Import'       
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccDigitalOceanKubernetesNodePool_Import -timeout 120m
?       github.com/terraform-providers/terraform-provider-digitalocean  [no test files]
=== RUN   TestAccDigitalOceanKubernetesNodePool_Import
--- FAIL: TestAccDigitalOceanKubernetesNodePool_Import (561.28s)
    testing.go:569: Step 1 error: 6 problems:
        
        - Cannot import non-existent remote object: While attempting to import an existing object to digitalocean_kubernetes_cluster.foobar-2, the provider detected that no object exists with the given id. Only pre-existing objects can be imported; check that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
        - Cannot import non-existent remote object: While attempting to import an existing object to digitalocean_kubernetes_cluster.foobar-6, the provider detected that no object exists with the given id. Only pre-existing objects can be imported; check that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
        - Cannot import non-existent remote object: While attempting to import an existing object to digitalocean_kubernetes_cluster.foobar-3, the provider detected that no object exists with the given id. Only pre-existing objects can be imported; check that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
        - Cannot import non-existent remote object: While attempting to import an existing object to digitalocean_kubernetes_cluster.foobar-4, the provider detected that no object exists with the given id. Only pre-existing objects can be imported; check that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
        - Cannot import non-existent remote object: While attempting to import an existing object to digitalocean_kubernetes_cluster.foobar-5, the provider detected that no object exists with the given id. Only pre-existing objects can be imported; check that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
        - Cannot import non-existent remote object: While attempting to import an existing object to digitalocean_kubernetes_cluster.foobar-1, the provider detected that no object exists with the given id. Only pre-existing objects can be imported; check that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
FAIL
FAIL    github.com/terraform-providers/terraform-provider-digitalocean/digitalocean     561.296s
FAIL
make: *** [GNUmakefile:18: testacc] Error 1
```

Thoughts?

Reply:
Commented inline on `resourceDigitalOceanKubernetesClusterImportState` with details.